index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="section-title">I'm Anderson Glad to see you!</h1>
     </section>
     <section id="works-section">
       <div class="work-container">
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a Figma design. Kindly update the `font-size` and spacing between elements to match the Figma template. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/206830372-7189a3a6-7bfb-4d85-937d-876499b21932.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/206830375-f750bcad-553a-461a-81ba-9ac57082568b.png" width="300">|

Reply:
Thanks for feedback. I will adapt the project to template